### PR TITLE
changed neo to talon

### PR DESCRIPTION
**Pull request by minhnguyenbhs**
I switched the CAN Spark Max intake motors to FalconFXs; is there anything else that needs to be changed?

### CODE REVIEW

File: vendordeps/AdvantageKit.json
Comment (aidnem) [inline vendordeps/AdvantageKit.json:1]: I assume updating vendordeps is fine and doesn't break anything?

**Review by aidnem — APPROVED** [review]
As long as updating the vendordeps is fine (I'll take your word for it), this looks good to me.

**Comment by jkleiber** [comment]
This looks good - let's test it on the robot before merging

File: vendordeps/photonlib.json
Comment (jkleiber) [inline vendordeps/photonlib.json:1]: Changes to this file result in code crash. Let's revert all vendor dependency upgrades that are not needed for this change

**Review by jkleiber — CHANGES_REQUESTED** [review]
Please revert all vendor dep updates unless required for the change

**Review by jkleiber — APPROVED** [review]
approved and will merge now

**Pull request merged**